submissions/sadoharu/task_js_dom_/scripts.js

```diff
@@ -0,0 +1,61 @@
+window.addEventListener("DOMContentLoaded", () => {
+  const pages = document.querySelectorAll(".section__page");
+  const navList = document.querySelector(".nav-bar__list");
+
+  const navListItem = [
+    {
+      title: "Overview",
+      icon: "icon-browser",
+    },
+    {
+      title: "About",
+      icon: "icon-about-dot-me",
+    },
+    {
+      title: "Achievement",
+      icon: "icon-award",
+    },
+    {
+      title: "Projects",
+      icon: "icon-files-empty",
+    },
+    {
+      title: "Reviews",
+      icon: "icon-rate_review",
+    },
+  ];
+
+  navListItem.forEach((item, i) => {
+    const liItem = document.createElement("li");
+    liItem.classList.add("nav-bar__list_item");
+    liItem.setAttribute("data-index", i);
+    liItem.innerHTML = `
+      <button class="nav-bar__list_button desktop">
+        <span class="${item.icon}">
+        </span>
+        ${item.title}
+      </button>
+      <button class="nav-bar__list_button mobile ${item.icon}"></button>
+    `;
+    navList.append(liItem);
+  });
+
+  navList.querySelector(".nav-bar__list_button").classList.add("active");
```

Comment:
Why not do this during creation of the list? 🤔

Reply:
i use forEach to create a list, and I only need one "active"

Reply:
So? What stops you from adding this property before adding your elements into DOM? 🤔

Reply:
I have them dynamically created 'li', how can i assign a class when i don't have 'li' created yet

Reply:
First of all, you controlling creation of them - so you have acces to strings in `forEach`. ;)
Second, you can create first item out of loop and then create others using forEach.
Third, you can create them, but not add to the DOM. Add your class and add elements to DOM after.

First option is better, as for me.